symplyphysics/laws/hydro/bernoullis_equation.py

```diff
@@ -0,0 +1,69 @@
+from sympy import Eq, solve, dsolve, Derivative
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Bernoulli's equation applied to an ideal liquid specifies that the sum of static,
+## dynamic and hydrostatic pressure is constant at all points along a streamline.
+
+# Law: d[P(t)/rho + v(t)**2/2 + g*h(t)]/dt = 0
```

Comment:
I suggest to write it the same way as it is described:
`d(P_static + P_dynamic + P_hydrostatic)/dt = 0`
Another option is to write it in even simplier form: `d(P_inner)/dt = 0`
And move each of referenced pressures to another law.

Reply:
This makes sense, I agree with your suggestion. We already have the necessary laws for dynamic and hydrostatic pressure, so it is possible to write this law in the simpler form. But I figure this would require a separate definition for what inner pressure in a fluid means, do you agree with that?

Reply:
Sure